diff_containerd.go

```diff
@@ -138,7 +139,7 @@ func doubleWalkDiff(ctx context.Context, changeFn ChangeFunc, a, b walkerFn, fil
 					return err
 				}
 				if f1.stat.IsDir() && !f2copy.stat.IsDir() {
-					rmdir = f1.path + string(os.PathSeparator)
+					rmdir = f1.path + string(filepath.Separator)
```

Comment:
(Just picked a random line); is all code always executed in the "native" environment, or are there also code-paths executed on non-matching platforms (client-side code running on Linux/mac on behalf of a daemon running on Windows, and vice-versa?)

Also note that `/` is accepted in various situations on Windows, so users may use `/` to specify paths on Windows, so in some cases _both_ `/` and `\` may have to be taken into account (if not normalised)